config/constants/settings/definitions.rb

```diff
@@ -922,8 +922,8 @@
 
   add :user_default_timezone,
       default: nil,
-      format: :timezone,
-      allowed: ActiveSupport::TimeZone.all + [nil]
+      format: :string,
+      allowed: ActiveSupport::TimeZone.all.map { |tz| tz.tzinfo.canonical_identifier } + [nil]
```

Comment:
```suggestion
      allowed: ActiveSupport::TimeZone.all.map { |tz| tz.tzinfo.canonical_identifier }.sort.uniq + [nil]
```

Add `sort.uniq` for a better error message when using invalid timezone name

Reply:
Good catch, thanks :+1: